### ISSUE TITLE
Can no longer add or edit URL's

### ISSUE THREAD

**Issue by 2braincells2go** [YOURLS/YOURLS#1887]
Been using Yourls for while and never had an issue. But all of a sudden can no longer add or edit URL's.

Once the "Shorten The URL" button is clicked just get the spinner. No error message and no messages in Chrome dev tools.

Basically same thing when attempting to edit existing link.

Hosting provider - of course says third party script and no issue there end.

Any ideas where I should starting looking to solve?

Server info:

Apache Version:	2.2.29
PHP Version:	5.4.39
MySQL Version:	5.5.42-cll

Many thanks in advance!

**Comment by ozh** [comment]
Rule of thumb: "all of a sudden" = something has changed, either within YOURLS config (plugin, something) or on the server side (apache, PHP, etc...)

Check wiki page https://github.com/YOURLS/YOURLS/wiki/Endless-Spinning-Icon and other issues about the same subject.

**Comment by 2braincells2go** [comment]
Very sorry to have bothered you with my issue.

Yes, I understand what all of a sudden means and have spent couple of days exploring what might have changed. Cannot seem to find anything.

Was reaching out to the community in hopes of help, but can see you closed such requests.

Cheers and wishing you a great day!

**Comment by ozh** [comment]
We close issues when they're not YOURLS defects, but they can still be useful to some and be commented on

What's your actual YOURLS install URL ?

**Comment by dylanh724** [comment]
Yea he does this. Not that Ozh will actually help you, but @2braincells2go , I suddenly ran across the same issue. At the same time I was looking into https://github.com/briancray/PHP-URL-Shortener so it doesn't matter that much to ME anymore, but just a heads up I'm getting the exact same thing. Thought it was a good time to switch.

I found it VERY odd that nothing was changed and all of a sudden .... I can't add or edit. The DB is fine or it wouldn't be able to load anything. CORS is fine or it would show in the console. Debug on shows nothing unusual.

This is quite a toxic community so I wouldn't expect answers, but ping me if I can provide more info to help from my end, reproducing this issue 100%.
